compiler/src/dotty/tools/dotc/parsing/xml/MarkupParsers.scala

```diff
@@ -277,7 +277,7 @@ object MarkupParsers {
      *                | xmlTag1 '/' '>'
      */
     def element: Tree = {
-      val start = curOffset
+      val start = curOffset // FIXME should be `curOffset - 1` (scalatest and tests/neg/i19100.scala must be updated)
```

Comment:
@EugeneFlesselle started working on updating the `scalatest` submodule. I will fix this once the submodule is updated to avoid conflicts.